src/ImageMagick.jl

```diff
@@ -115,39 +118,22 @@ function load_(file::Union{AbstractString,IO,Vector{UInt8}}; ImageType=Image, ex
         end
     end
     # Allocate the buffer and get the pixel data
-    buf = Array(T, sz...)
+    buf = Array{T}(sz)
```

Comment:
No, I just meant they are multidimensional, not one-dimensional. Some TIFF files, for example, have a 3rd dimension (and that's properly handled here).